Android/ConstraintLayout/source/ConstraintLayout/ConstraintLayout.csproj

```diff
@@ -44,6 +45,45 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
     <Reference Include="Mono.Android" />
+    <Reference Include="Xamarin.Android.Support.Annotations">
+      <HintPath>..\..\samples\packages\Xamarin.Android.Support.Annotations.28.0.0-preview7\lib\monoandroid81\Xamarin.Android.Support.Annotations.dll</HintPath>

```

Comment:
Let's convert this puppy to PackageReference

Reply:
Done